### DISCUSSION TITLE
Playing a common part from the server

### DISCUSSION THREAD

**Discussion by MaxYaffe** [jamulussoftware/jamulus#2295] [General]
I apologize if this is already asked but I may not know what terms to search for.
I'm trying to set up a virtual choir on Jamulus.  Is it possible to have a server play a common accompaniment part so that all the remote singers hear from the same source?  Of course I'm assuming the singers all have similar delays to that server.

**Comment by pljones** [comment]
Yes - have the audio stream from a client connected to the server in use.  Windows, Linux and MacOS all support means of doing this, although it's easier on some platforms than others.

**Reply by hoffie** [reply]
Yep, I've been doing exactly that with my choir multiple times.
Jamulus-wise, this is just a normal client (could probably even be headless?). If you can, host it close to the server (it was running on the same physical host as the server in our sertup). Then, connect the playback (mplayer, vlc, or maybe even a browser) to Jamulus' input.
This works out of the box on linux (via qjackctl and its routing possibilities).

Examples for other platforms are probably VoiceMeeter (Banana) for Windows or BlackHole for macOS.

**Reply by MaxYaffe** [reply]
Thanks, Hoffie.  What is a headless client?  I assume that's one without a UI but is it on a separate machine, or a separate process on the same machine?  I'll look into VoiceMeeter.

**Reply by henkdegroot** [reply]
You might be interested in this KB: [How to play audio from your PC to Jamulus (using JACK)](https://jamulus.io/kb/2022/01/26/Routing-JACK-Windows.html)

It may have some more details you can use.

**Reply by ann0see** [reply]
Yeah. And if there are any issues with the article, feel free to edit/comment the file
